### PR TITLE
update docs to reflect CheckoutWithNativeWebView -> CheckoutWithWebView

### PR DESCRIPTION
**Pull request by richardmonette**
Updates the docs, etc. since we've changed the method name.

@Shopify/gaming @coreypollock

### CODE REVIEW

File: EXAMPLES.md [hunk 1]
```diff
@@ -252,10 +252,10 @@ cart.LineItems.Delete(firstProduct, selectedOptions);
 
 ## Native web view checkout
 
-After creating an instance of `Cart` and adding items to it, you can use the `CheckoutWithNativeWebView` method to
+After creating an instance of `Cart` and adding items to it, you can use the `CheckoutWithWebView` method to
 start a native modal overlay on top of your game with a web view that contains the checkout for the cart.
```
Comment (dengjeffrey) [inline EXAMPLES.md]: `start a modal overlay`?

File: scripts/generator/graphql_generator/csharp/Cart.cs.erb [hunk 2]
```diff
@@ -146,7 +146,7 @@ namespace <%= namespace %> {
 
         /// <summary>
         /// Returns the web URL for checking out the contents of this Cart. For presenting the user with a web view that
-        /// loads this URL see <see ref="Cart.CheckoutWithNativeWebView">CheckoutWithNativeWebView</see>.
+        /// loads this URL see <see ref="Cart.CheckoutWithWebView">CheckoutWithWebView</see>.
```
Comment (dengjeffrey) [inline scripts/generator/graphql_generator/csharp/Cart.cs.erb]: Realized this is wrong, should be in the format of: `<see cref="member"/>`

File: EXAMPLES.md [hunk 3]
```diff
@@ -275,7 +275,7 @@ ShopifyBuy.Client().products((products, error) => {
     cart.LineItems.AddOrUpdate(productVariantToCheckout, 1);
 
     // Launches the native web checkout experience overlaid on top of your game.
```
Comment (dengjeffrey) [inline EXAMPLES.md]: `// Launches the web checkout experience overlaid on top of your game.`

File: docs/index.html [hunk 4]
```diff
@@ -110,14 +110,14 @@
...
 <div class="fragment"><div class="line">cart.LineItems.Get(firstProduct, selectedOptions);</div><div class="line">cart.LineItems.Delete(firstProduct, selectedOptions);</div></div><!-- fragment --><h2>Native web view checkout</h2>
```
Comment (dengjeffrey) [inline docs/index.html]: Seems like there are still some places that we missed that still reference a `Native web view checkout`

**Comment by richardmonette** [comment]
@dengjeffrey updated!

File: docs/index.html [hunk 5]
```diff
@@ -109,17 +109,17 @@
...
-<div class="fragment"><div class="line">cart.LineItems.Get(firstProduct, selectedOptions);</div><div class="line">cart.LineItems.Delete(firstProduct, selectedOptions);</div></div><!-- fragment --><h2>Native web view checkout</h2>
-<p>After creating an instance of <code>Cart</code> and adding items to it, you can use the <code>CheckoutWithNativeWebView</code> method to start a native modal overlay on top of your game with a web view that contains the checkout for the cart.</p>
-<p><code>CheckoutWithNativeWebView</code> takes in 3 callback parameters:</p>
+<div class="fragment"><div class="line">cart.LineItems.Get(firstProduct, selectedOptions);</div><div class="line">cart.LineItems.Delete(firstProduct, selectedOptions);</div></div><!-- fragment --><h2>Web view checkout</h2>
+<p>After creating an instance of <code>Cart</code> and adding items to it, you can use the <code>CheckoutWithWebView</code> method to start a native modal overlay on top of your game with a web view that contains the checkout for the cart.</p>
```
Comment (dengjeffrey) [inline docs/index.html]: Just this last description needs to change: `method to start a native modal overlay on top of your game with a web view that contains the checkout for the cart`

It's no longer a native modal overlay or a web view

**Pull request merged**